src/VisualStudio/Core/Def/Implementation/FindReferences/Entries/MetadataDefinitionItemEntry.cs

```diff
@@ -27,8 +28,14 @@ private class MetadataDefinitionItemEntry : AbstractItemEntry, ISupportsNavigati
             {
                 switch (keyName)
                 {
+                    case StandardTableKeyNames.ProjectName:
+                        return DefinitionBucket.DefinitionItem.OriginationParts.JoinText();
+                    case StandardTableKeyNames.DocumentName:
+                        return DefinitionBucket.DefinitionItem.Properties[AbstractReferenceFinder.ContainingTypeInfoPropertyName];
```

Comment:
can you clarify what you'll see for ProjectName?

Reply:
`OriginationParts` is the containing project or assembly name: https://github.com/dotnet/roslyn/blob/e2d317dff5437937b6aa7d90a7f1af0dd1b0906a/src/Features/Core/Portable/FindUsages/DefinitionItem.cs#L86-L90`